Source/SwiftLintFramework/Configuration/Configuration+Remote.swift

```diff
@@ -86,7 +86,10 @@ internal extension Configuration.FileGraph.FilePath {
             // Block main thread until timeout is reached / task is done
             while true {
                 if taskDone { break }
-                if Date().timeIntervalSince(startDate) > timeout { task.cancel(); break }
+                if Date().timeIntervalSince(startDate) > timeout {
+                    task.cancel()
+                    break
+                }
```

Comment:
This doesn't look it got auto-fixed by the rule. The only cases that can be fixed automatically are the ones where the statements are on their own line without leading tokens. We should probably restrict the rewriter to these cases.